addOns/pscanrules/src/main/java/org/zaproxy/zap/extension/pscanrules/TimestampDisclosureScanRule.java

```diff
@@ -123,10 +123,13 @@ public String getName() {
 
     @Override
     public void scanHttpResponseReceive(HttpMessage msg, int id, Source source) {
-        if (ResourceIdentificationUtils.isFont(msg)
-                || ResourceIdentificationUtils.isJavaScript(msg)) {
+        if (ResourceIdentificationUtils.isFont(msg)) {
             return;
         }
+        if (this.getAlertThreshold() == AlertThreshold.HIGH && ResourceIdentificationUtils.isJavaScript(msg)){
+           LOGGER.debug("Skipping Javascript files from TimestampDisclosureScanRule as alert threshold is set to High");
+           return;
+        }
```

Comment:
These could be combined. Once you've done some live testing the logging can be removed.

Also I believe enum comparison should use `.equals()` ex: `AlertThreshold.HIGH.equals(this.getAlertThreshold())`

Reply:
Do you want to skip font files only when threshold is high?

The comparison style is inspired from https://github.com/zaproxy/zap-extensions/blob/124016cff14eb93673c390353251103fd2bbd2c9/addOns/ascanrules/src/main/java/org/zaproxy/zap/extension/ascanrules/SourceCodeDisclosureCve20121823ScanRule.java#L133